lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml

```diff
@@ -1253,6 +1285,20 @@ states:
       access-denied:
         targetJourney: FAILED
         targetState: FAILED
+        checkFeatureFlag:
+          f2fVolumesJourneyEnabled:
+            targetState: MITIGATION_PP_PROVE_ANOTHER_WAY
+
+  MITIGATION_PP_PROVE_ANOTHER_WAY:
+    response:
+      type: page
+      pageId: prove-identity-no-other-photo-id
+      context: passport
+    events:
+      back:
+        targetState: MITIGATION_PP_CRI_UK_PASSPORT
+      returnToRp:
+        targetState: RETURN_TO_RP
```

Comment:
The returnToRp events where a user is mitigationg will also need to go to TICF. I think you might need to create a new state similar to `INELIGIBLE_SKIP_MESSAGE`, but in the failed sub journey.

Reply:
Would INELIGIBLE/INELIGIBLE_SKIP_MESSAGE not work? I see the state already fails if there's a ci - which is what we'd want since there's a doubling-up.

Reply:
I think it's about the difference between being ineligble and failing. In this scenario the user has a CI they can't mitigate, so they've failed.